.github/workflows/pr-bot-welcome.yml

```
@@ -0,0 +1,28 @@
+name: PR Welcome Bot
+
+on:
+  pull_request:
```

I'm not sure if `pull_request` can request a token with these permissions from forks

---

Correct! I forgot to update this PR
```suggestion
  pull_request_target:
```